adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -139,7 +139,7 @@ workflow AdapterOptimus {
   Int max_cromwell_retries = 0
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.47.0"
+  String pipeline_tools_version = "rex-metadata-schema-update"
```

Comment:
this temporary for dev purposes?

Reply:
Yeah, due to the nature of this repo (a mixture of dockerized Python package and WDLs that refer to the package in the command blocks), every time we modify the Python package in this repo, we have to follow this special process:
- Change the Python package and use the `branch_name` for all of the `pipeline_tools_version` that were previously semantic (e.g. `v0.47.0`)
- Get approvals on your PR
- Replace the `branch_name` with `semantic version + 1` (e.g.  `v0.48.0`)
- Merge the PR and release the repo/Python Package with `semantic version`. (e.g.  `v0.48.0`)